Dalamud/Interface/Internal/Windows/Settings/Tabs/SettingsTabExperimental.cs

```diff
@@ -55,6 +70,45 @@ public class SettingsTabExperimental : SettingsTab
 
         new ThirdRepoSettingsEntry(),
 
+        new GapSettingsEntry(5, true),
+
+        new EnumSettingsEntry<ReShadeHandlingMode>(
+            Loc.Localize("DalamudSettingsReShadeHandlingMode", "ReShade handling mode"),
+            Loc.Localize(
+                "DalamudSettingsReShadeHandlingModeHint",
+                "You may try different options to work around problems you may encounter.\nRestart is required for changes to take effect."),
+            c => c.ReShadeHandlingMode,
+            (v, c) => c.ReShadeHandlingMode = v,
+            fallbackValue: ReShadeHandlingMode.ReShadeAddon)
+        {
+            FriendlyEnumNameGetter = x => x switch
+            {
+                ReShadeHandlingMode.ReShadeAddon => Loc.Localize(
+                    "DalamudSettingsReShadeHandlingModeReShadeAddon",
+                    "ReShade addon"),
+                ReShadeHandlingMode.UnwrapReShade => Loc.Localize(
+                    "DalamudSettingsReShadeHandlingModeUnwrapReShade",
+                    "Unwrap ReShade"),
+                ReShadeHandlingMode.None => Loc.Localize(
+                    "DalamudSettingsReShadeHandlingModeNone",
+                    "Do not handle"),
+                _ => "<invalid>",
+            },
+            FriendlyEnumDescriptionGetter = x => x switch
+            {
+                ReShadeHandlingMode.ReShadeAddon => Loc.Localize(
+                    "DalamudSettingsReShadeHandlingModeReShadeAddonDescription",
+                    "Dalamud will register itself as a ReShade addon. Most compatibility is expected, but multi-monitor window option won't work too well."),
```

Comment:
Could you describe what "won't work too well" means? Will you be able to use it at all/will it be affected by shaders?

Reply:
Viewport windows worked fine for me with ReShade, but every time it gets created it did have to load all over again.